grpc/bcodes.go

```diff
@@ -83,10 +103,35 @@ func wrapError(err error) error {
 	return grpc.Errorf(code, body)
 }
 
-func unwrapError(err error) error {
+// unwrapError unwraps errors returned from gRPC client calls which were wrapped
+// with wrapError to their proper internal error type. If the error is a
+// BoulderError the grpc/metadata taken from the call context.Context is used to
```

Comment:
This is not really accurate. Instead I would say "If the provided metadata object has an "errortype" field, that will be used to set the type of the error.